src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -59,6 +66,27 @@ if(coreHasFeature(REFERENCE_ICON_USING_STRING)) {
     expect(vm.$el.tagName).toBe('svg')
     expect(vm.$el.classList.contains('fa-coffee')).toBeTruthy()
   })
+
+  test('using a FASS icon with string format', () => {
+    const vm = mountFromProps({ icon: 'fass fa-arrow-left'})
+  
+    expect(vm.$el.tagName).toBe('svg')
+    expect(vm.$el.classList.contains('fa-arrow-left')).toBeTruthy()
+  })
+
+  test('using a sharp solid icon long prefix with array format', () => {
+    const vm = mountFromProps({ icon: 'fa-sharp fa-arrow-left' })
```

Comment:
```suggestion
    const vm = mountFromProps({ icon: 'fa-sharp fa-solid fa-arrow-left' })
```

do we also need `fa-solid` here?

Reply:
No. We want to support `'fa-sharp fa-icon` syntax here and it will default to sharp solid kind of like how we default to solid when just calling `'fa-icon'`.